### PR TITLE
Harpy Customization Pack 1

### PR DESCRIPTION
**Pull request by VMSolidus**
## About the PR
This is the first of the promised content PRs for harpies, and also includes some bugfixes. 

## Why / Balance
-5 new wing patterns are added. Two Tone, Three Tone, Speckled, Undertone, and Feather Tips
-The HUD bug has been fixed
-Sparrow Tail has been removed for REASONS. 
-Harpies can now choose what instrument they want to imitate(From within a list of options). Presumably this means multiple harpies can start a band together! Right click yourself to select what instrument you'll mimic. 
## Media

New customization
swappableharpies

### CODE REVIEW

**Comment by Colin-Tel** [comment]
image
The "Phoenix Tail" seems to be the default tail, the only reason a player would add it again is if they want to change the color of the tail. It comes across as a bug-at-first-sight, which I think could be addressed by just calling it "Basic Tail" much like the "Basic Wings"

**Comment by Colin-Tel** [comment]
image
There's also missing localization strings when it comes to *all* harpy markings.

**Comment by evilexecutive** [comment]
I've made the requested changes.

File: Content.Server/DeltaV/Harpy/HarpySingerSystem.cs
Comment (DebugOk) [inline Content.Server/DeltaV/Harpy/HarpySingerSystem.cs:1]: Why subscribe to the same event twice

**Review by DebugOk — CHANGES_REQUESTED** [review]

File: Content.Server/DeltaV/Harpy/HarpySingerSystem.cs
Comment (evilexecutive) [reply Content.Server/DeltaV/Harpy/HarpySingerSystem.cs:1]: Let's hope the rogue semicolon doesn't strike again

**Comment by DebugOk** [comment]
I still like how the customization code makes markings flesh colored by default. Flesh wings!

**Comment by DebugOk** [comment]
> image There's also missing localization strings when it comes to _all_ harpy markings.

This is a bug:tm: unrelated to harpies (probably)

I also saw it on moths

**Comment by evilexecutive** [comment]
>This is a bug™️ unrelated to harpies (probably)
>I also saw it on moths

Oh it's not, check the localization files on this PR for the fix. You can also apply that to moths

**Review by DebugOk — DISMISSED** [review]
Code looks good enough, the rest Colin can review

File: Content.Server/DeltaV/Harpy/HarpySingerSystem.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+using Content.Shared.Mobs.Systems;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.DeltaV.Harpy
+{
+    public sealed class HarpySingerSystem : EntitySystem
+    {
+        [Dependency] private readonly InstrumentSystem _instrument = default!;
+        [Dependency] private readonly MobStateSystem _mobState = default!;
+        public override void Initialize()
```
Comment (DEATHB4DEFEAT) [inline Content.Server/DeltaV/Harpy/HarpySingerSystem.cs:12]:
```suggestion
        [Dependency] private readonly InstrumentSystem _instrument = default!;
        [Dependency] private readonly MobStateSystem _mobState = default!;


        public override void Initialize()
```

File: Content.Server/DeltaV/Harpy/HarpySingerSystem.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<InstrumentComponent, MobStateChangedEvent>(HarpyStopSinging);
+        }
+        //Immediately closes the Midi UI window if a Singer is incapacitated
+        private void HarpyStopSinging(EntityUid uid, InstrumentComponent component, MobStateChangedEvent args)
+
+        {
```
Comment (DEATHB4DEFEAT) [inline Content.Server/DeltaV/Harpy/HarpySingerSystem.cs:21]:
```suggestion
        }


        //Immediately closes the Midi UI window if a Singer is incapacitated
        private void HarpyStopSinging(EntityUid uid, InstrumentComponent component, MobStateChangedEvent args)
        {
```

**Review by DEATHB4DEFEAT — COMMENTED** [review]
muh formatting

**Review by Colin-Tel — APPROVED** [review]
🐦

**Pull request merged**